vb_suite/indexing.py

```diff
@@ -15,9 +15,8 @@
 dt = ts.index[500]
 """
 statement = "ts[dt]"
-
 bm_getitem = Benchmark(statement, setup, ncalls=100000,
-                       name='series_getitem_scalar')
+                       name='time_series_getitem_scalar')
```

Comment:
This was same Benchmark name as L220 one. This doesn't cause error, but confusing.